bin/PythonApps/EPLaunch.app/Contents/MacOS/EPLaunch

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+SCRIPT_DIR="$( cd -- "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 ; pwd -P )"
+echo "Script Path: $SCRIPT_DIR"
```

Comment:
This is just a debug when you run it directly in bash via
`./EPLaunch.app/Contents/MacOS/EPLaunch` and not `open EPLaunch.app`